content.js

```diff
@@ -142,29 +196,35 @@ if (checks()) {
 			case '__PSZY_SWAP__':
 				moveswap(e.target.parentNode.parentNode)
 				break
+			case '__PSZY_MOVETO__':
+				moveto(e.target.parentNode.parentNode)
+				break
 			case '__PSZY_MOVERANGE__':
 				moverange(e.target.parentNode.parentNode)
 				break
 			case '__PSZY_PBANK__':
-				if(temp===0){
-					iframer(e.target.parentNode.parentNode)
-					temp=1
-				}else{
-					iframeremover(e.target.parentNode.parentNode)
-					temp=0
-				}
+				let stid = e.target.parentNode.parentNode.querySelector('.spanclass.uiicon').attributes.spn.value
+				fetch("http://psd.bits-pilani.ac.in/Student/ViewActiveStationProblemBankData.aspx/getPBPOPUP", {
+					"headers": {
+						"accept": "application/json, text/javascript, */*; q=0.01",
+						"accept-language": "en-US,en;q=0.9",
+						"cache-control": "no-cache",
+						"content-type": "application/json; charset=UTF-8",
+						"pragma": "no-cache",
+						"x-requested-with": "XMLHttpRequest"
+					},
+					"referrer": "http://psd.bits-pilani.ac.in/Student/ViewActiveStationProblemBankData.aspx",
+					"referrerPolicy": "strict-origin-when-cross-origin",
+					"body": `{StationId: \"${stid}\" }`,
```

Comment:
Also, the JSON key `StationId` is technically wrong because it's not surrounded by quotes, I'm not sure how PSD backend is able to parse this as valid JSON